monitoring/snippets/v3/uptime-check-client/snippets_test.py

```diff
@@ -87,19 +87,19 @@ def call_sample():
         assert new_uptime_check_path in out
 
 
-def test_get_uptime_check_config(capsys, uptime):
+def test_get_uptime_check_config(capsys, uptime) -> None:
     snippets.get_uptime_check_config(uptime.config_get.name)
     out, _ = capsys.readouterr()
     assert uptime.config_get.display_name in out
```

Comment:
Again, change all the functions being tested to return a value, and write assertions about that returned value, not about captured text.

Reply:
done